kubectl-fdb/cmd/remove_instances.go

```diff
@@ -0,0 +1,205 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/FoundationDB/fdb-kubernetes-operator/controllers"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ctx "context"
+
+	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
+)
+
+func newRemoveInstancesCmd(streams genericclioptions.IOStreams, rootCmd *cobra.Command) *cobra.Command {
+	o := NewFDBOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:   "instances",
+		Short: "Adds an instance (or multiple) to the remove list of the given cluster",
+		Long:  "Adds an instance (or multiple) to the remove list field of the given cluster",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			force, err := rootCmd.Flags().GetBool("force")
+			if err != nil {
+				return err
+			}
+			cluster, err := cmd.Flags().GetString("fdb-cluster")
+			if err != nil {
+				return err
+			}
+			withExclusion, err := cmd.Flags().GetBool("exclusion")
+			if err != nil {
+				return err
+			}
+			withShrink, err := cmd.Flags().GetBool("shrink")
+			if err != nil {
+				return err
+			}
+			useInstanceID, err := cmd.Flags().GetBool("use-instance-id")
+			if err != nil {
+				return err
+			}
+
+			config, err := o.configFlags.ToRESTConfig()
+			if err != nil {
+				return err
+			}
+
+			scheme := runtime.NewScheme()
+			_ = clientgoscheme.AddToScheme(scheme)
+			_ = fdbtypes.AddToScheme(scheme)
+
+			kubeClient, err := client.New(config, client.Options{Scheme: scheme})
+			if err != nil {
+				return err
+			}
+
+			namespace, err := getNamespace(*o.configFlags.Namespace)
+			if err != nil {
+				return err
+			}
+
+			instances := args
+			if !useInstanceID {
+				instances, err = getInstanceIDsFromPod(kubeClient, cluster, instances, namespace)
+				if err != nil {
+					return err
+				}
+			}
+
+			return removeInstances(kubeClient, cluster, instances, namespace, withExclusion, withShrink, force)
+		},
+		Example: `
+# Remove instances for a cluster in the current namespace
+kubectl fdb remove instances -c cluster instance-1 -i instance-2
+
+# Remove instances for a cluster in the namespace default
+kubectl fdb -n default remove instances -c cluster instance-1 instance-2
+
+# Remove instances for a cluster with the instance-id
+kubectl fdb -n default remove instances --use-instance-id -c cluster storage-1 storage-2
+`,
+	}
+
+	cmd.Flags().StringP("fdb-cluster", "c", "", "remove instance(s) from the provided cluster.")
+	cmd.Flags().BoolP("exclusion", "e", true, "define if the instances should be remove with exclusion.")
+	cmd.Flags().Bool("use-instance-id", false, "if set the operator will use the instance ID to remove it rather than the Pod name.")
```

Comment:
I suggest replacing "it" with "pod(s)", as this command may process multiple pods. 

Also Pod here should be lower cased, or Instances should be upper cased for consistency when talking about types, I think?